boards/arduino-mkr1000/Kconfig

```
@@ -0,0 +1,15 @@
+# Copyright (c) 2020 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+source "$(RIOTBOARD)/common/arduino-mkr/Kconfig"
+
+config BOARD
+    default "arduino-mkr1000" if BOARD_ARDUINO_MKR1000
+
+config BOARD_ARDUINO_MKR1000
+    bool
+    default y
+    select BOARDS_COMMON_ARDUINO-MKR
```

`-` can't be used in the symbol name
```suggestion
    select BOARDS_COMMON_ARDUINO_MKR
```